### PR TITLE
🐛 Add missing help docs

### PR DESCRIPTION
**Pull request by tas50**
This syncs with cnquery

### CODE REVIEW

File: apps/cnspec/cmd/bundle.go [hunk 1]
```diff
@@ -129,7 +129,7 @@ var policyLintCmd = &cobra.Command{
 var policyFmtCmd = &cobra.Command{
 	Use:     "format [path]",
 	Aliases: []string{"fmt"},
-	Short:   "Apply style formatting to policy bundles.",
+	Short:   "Apply style formatting to a policy bundle.",
```
Comment (czunker) [inline apps/cnspec/cmd/bundle.go]: Didn't know this, so I gave it a try. The command can actually scan multiple bundles:
```
cnspec bundle lint ~/workspace/mondoo/github.com/cnspec-policies/community/ --verbose                                                                            ✔ │ 06:15:20 
→ lint policy bundle file=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/chef-infra-client.mql.yaml
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/chef-infra-server.mql.yaml
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/mondoo-linux-operational-policy.mql.yaml
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/mondoo-linux-snmp-policy.mql.yaml
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/mondoo-phoenix-plcnext-security.mql.yaml
→ valid policy bundle
```

Should we keep the text as it was?

File: apps/cnspec/cmd/bundle.go [hunk 2]
```diff
@@ -214,7 +214,7 @@ var policyPublishCmd = &cobra.Command{
 		}
 		queryHubServices, err := policy.NewPolicyHubClient(opts.UpstreamApiEndpoint(), httpClient, certAuth)
 		if err != nil {
-			log.Fatal().Err(err).Msg("could not connect to policy hub")
+			log.Fatal().Err(err).Msg("could not connect to Security Registry")
```
Comment (czunker) [inline apps/cnspec/cmd/bundle.go]: The other changes use `the Mondoo Security Registry`. Should we make it consistent?

File: apps/cnspec/cmd/run.go [hunk 3]
```diff
@@ -17,8 +18,147 @@ func init() {
...
+				Short: "Run an MQL query against a specific AWS snapshot using an EBS volume scan. This requires an AWS host.",
+			},
+			"aws-ec2-ssm": {
+				Short: "Run an MQL query against an AWS instance using the AWS Systems Manager to connect.",
+			},
+			"azure": {
+				Short: "Run an MQL query against a Microsoft Azure subscription or virtual machine.",
+			},
+			"gcp": {
+				Short: "Run an MQL query against a Google Cloud Platform (GCP) organization, project or folder.",
```
Comment (czunker) [inline apps/cnspec/cmd/run.go]: I think, this is missing:
```suggestion
				Short: "Run an MQL query against a Google Cloud Platform (GCP) organization, project, or folder.",
```

File: apps/cnspec/cmd/run.go [hunk 4]
```diff
@@ -17,8 +18,147 @@ func init() {
...
+				Short: "Run an MQL query against a Google Cloud Platform (GCP) folder.",
+			},
+			"gcp-gcr": {
+				Short: "Run an MQL query against a Google Container Registry (GCR).",
+			},
+			"gcp-compute-instance": {
+				Short: "Run an MQL query against a Google Cloud Platform (GCP) VM instance.",
+			},
+			"oci": {
+				Short: "Run an MQL query against a Oracle Cloud Infrastructure (OCI) tenancy.",
```
Comment (czunker) [inline apps/cnspec/cmd/run.go]:
```suggestion
				Short: "Run an MQL query against an Oracle Cloud Infrastructure (OCI) tenancy.",
```

File: apps/cnspec/cmd/shell.go [hunk 5]
```diff
@@ -146,31 +147,46 @@ You can also specify a subcommand to narrow the scan to containers or images.
...
 				Short: "Connect to an AWS instance using an EBS volume scan. This requires an AWS host.",
-				Long:  `Connect to an AWS instance using an EBS volume scan. using an EBS volume scan. This requires that the scan execute on an instance that is running inside of AWS.`,
+				Long: `Connect to an AWS instance using an EBS volume scan. This requires that the
+scan execute on an instance that is running inside of AWS.`,
 			},
 			"aws-ec2-ebs-volume": {
 				Short: "Connect to a specific AWS volume using an EBS volume scan. This requires an AWS host.",
-				Long:  `Connect to a specific AWS volume using the EBS volume scan capability. This requires that the scan execute on an instance that is running inside of AWS.`,
+				Long: `Connect to a specific AWS volume using an EBS volume scan. This requires that the
+				scan execute on an instance that is running inside of AWS.`,
```
Comment (czunker) [inline apps/cnspec/cmd/shell.go]: indent wrong?

File: apps/cnspec/cmd/shell.go [hunk 6]
```diff
@@ -146,31 +147,46 @@ You can also specify a subcommand to narrow the scan to containers or images.
...
 			},
 			"aws-ec2-ebs-snapshot": {
 				Short: "Connect to a specific AWS snapshot using an EBS volume scan. This requires an AWS host.",
-				Long:  `Connect a specific AWS snapshot using the EBS volume scan capability. This requires that the scan execute on an instance that is running inside of AWS.`,
+				Long: `Connect to a specific AWS snapshot using an EBS volume scan. This requires that the
+				scan execute on an instance that is running inside of AWS.`,
```
Comment (czunker) [inline apps/cnspec/cmd/shell.go]: indent wrong?

**Review by czunker — APPROVED** [review]
Nice work. Thanks @tas50 

I only found some minor nit picks.

File: apps/cnspec/cmd/bundle.go (hunk 1 above)
Comment (tas50) [reply apps/cnspec/cmd/bundle.go]: Good call. Let me fix that tomorrow

File: apps/cnspec/cmd/bundle.go (hunk 2 above)
Comment (tas50) [reply apps/cnspec/cmd/bundle.go]: Might as well

**Pull request merged**